eip155/README.md

```diff
@@ -0,0 +1,40 @@
+---
+namespace-identifier: eip155
+title: EIP155 Namespace, aka EVM Chains
+---
+
+# Namespace for EIP155 chains
+
+This document describes the syntax and structure of the [EIP155][] namespace,
+commonly referred to as the "EVM" namespace for chains using the Ethereum
+Virtual Machine. 
+
+## Introduction
+
+[EIP155][] is an Ethereum Improvement Process specifying addressing across
+Ethereum-based chains, generalized by [CAIP-2][]. Definition of valid
+`references` within that namespace and constraints are deferred to the
+[EIP155][] specification, which may be superseded by future EIPs.
+
+## References
+
+- [EIP155][]: Ethereum Improvement Proposal specifying generation and validation of ChainIDs
+- [ethereum-lists/chains][]: An open registry for eip155 network operators to claim a
+      unique chainID and self-publish RPC/node information for them.
+- [ERC20][]: Basic [aka Fungible] Token Standard
+- [ERC721][]: Non-Fungible Token Standard
+
+[Chainid.network]: https://github.com/ethereum-lists/chains
```

Comment:
```suggestion
[Chainid.network](https://github.com/ethereum-lists/chains)
```

Reply:
check it in github, this is a cool GH-flavored markdown trick that cel taught me!

Reply:
Oh, I see. That's confusing :)